src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -20,6 +20,11 @@ export type ButtonGroupProps = {
    * and returns the `ReactNode` to render.
    */
   overflowButton?: (firstOverflowingIndex: number) => React.ReactNode;
```

Comment:
Maybe change `overflowButton` description a bit

Reply:
Changed and added info about the new prop, which made me realize maybe the prop should be called `overflowPlacement`

Reply:
Ok, lets wait for others opinion about the naming 😀

Reply:
I would have gone with `position`, but only because we have tippy in multiple components and its using placement, `placement` feels like more consistent one...

Reply:
![image](https://user-images.githubusercontent.com/9084735/152982874-24f4c086-1350-477d-936d-87b83f2366cf.png)